torch_xla/csrc/runtime/env_vars.cc

```diff
@@ -16,6 +16,7 @@ const char* const kEnvPjrtAsyncGpuClient = "PJRT_GPU_ASYNC_CLIENT";
 const char* const kEnvTpuLibraryPath = "TPU_LIBRARY_PATH";
 const char* const kEnvXpuLibraryPath = "XPU_LIBRARY_PATH";
 const char* const kEnvPjrtDistServiceAddr = "PJRT_DIST_SERVICE_ADDR";
+const char* const kEnvPjRtLocalProcessCount = "PJRT_LOCAL_PROCESS_COUNT";
```

Comment:
can we add this to https://github.com/pytorch/xla/blob/master/configuration.yaml

Reply:
This var shouldn't be expected to be set by users, so I guess we don't export to the configuration.yaml? It's set here https://github.com/pytorch/xla/blob/74d115ec778c2463ca422d800a0948bc62780f20/torch_xla/_internal/pjrt.py#L109